### ISSUE TITLE
Force Push fails for css (duplicate value found: AuraDefinitionBundleId duplicates value on record with id)

### ISSUE THREAD

**Issue by synle** [ForceCLI/force#403]
The first time it pushes the css file correctly, but trying to do it again throws error

```
force aura push -f src/aura/ObjectCard/
  Updating ObjectCard.cmp done
  Updating ObjectCard.css ERROR: duplicate value found: AuraDefinitionBundleId duplicates value on record with id: <some_ids>
%!(EXTRA string=DUPLICATE_VALUE)
```

**Comment by dscottwarren** [comment]
I am also experiencing this issue. Pushing individual files (other than .css) works fine. However, pushing the bundle or the .css file individually results in the error:

```
"ERROR: duplicate value found: AuraDefinitionBundleId duplicates value on record with id: ###############
%!(EXTRA string=DUPLICATE_VALUE)"
```

**Comment by dcarroll** [comment]
Do you have a .manifest file in the component bundle folder?  Did you try
to rename the bundle?

**Comment by dscottwarren** [comment]
Thanks for the response @dcarroll!

There is a .manifest file in my bundle, and no files were renamed as part of the push. 

Generally speaking, I run a fetch against a package.xml file, then "push" the component (just the .cmp file, not the bundle as the bundle fails with the CSS error) in order to create the .manifest file for the first time. Then I push individual files as they are modified. However, pushing the bundle or the CSS file by itself results in the error message.

**Comment by cwarden** [comment]
Should be fixed by #657.  `.manifest` files are no longer used.
